### PR TITLE
fix: Build folder cleanup

### PR DESCRIPTION
**Pull request by Jklawreszuk**
# PR Details

PR removes unused files located in build folder. I also fixed BuildXmlDocumentation.bat file

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] **I have built and run the editor to try this change out.**

### CODE REVIEW

**Comment by Jklawreszuk** [comment]
To speed up the review, I would like to add some comments on my changes

File: build/BuildXmlDocumentation.bat
Comment (Jklawreszuk) [inline build/BuildXmlDocumentation.bat:1]: `StrideGenerateDoc` was change into `StridePublicApi` (https://github.com/stride3d/stride/commit/2c3f881b0008fb63dede4664b7b8ba810af3390d), seems to forgot to add a change here

File: build/BuildXmlDocumentationVS2015.bat
Comment (Jklawreszuk) [inline build/BuildXmlDocumentationVS2015.bat:1]: duplicate

File: build/KillMSBuild.bat
Comment (Jklawreszuk) [inline build/KillMSBuild.bat:1]: The script is not used anywhere.The script itself is too simple to be in the source code better to save in user gist

File: build/NugetExtractFix.ps1
Comment (Jklawreszuk) [inline build/NugetExtractFix.ps1:1]: The script was used in the XenkoSample.build file (https://github.com/stride3d/stride/commit/53ce6d3433cf02c51921e673afd3430177c2bbcc), but was simplied, probably in favor of StrideSamples.sln; unused

**Comment by Eideren** [comment]
Cool, thanks a bunch for the notes

**Pull request merged**